### PR TITLE
Completed javascript-if exercise.

### PR DESCRIPTION
**Pull request by EuichangPark**
javascript-if

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+function isOldEnoughToDrinkAndDrive(person) {
+  let result;
+  if (person.age > 21) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js:68]: This should always return `false`:

---

image

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    result = 'acid';
+  } else if (pH === 7) {
+    result = 'neutral';
+  } else if (pH > 7 && pH < 14) {
+    result = 'base';
+  } else {
+    result = 'invalid pH level';
+  }
+  return result;
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js:82]: `0` and `14` are valid `pH` levels, so they should return `'acid'` and `'base'` (respectively):

---

image

File: javascript-if/index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (TimelikeClosure) [inline javascript-if/index.html:7]: Please `<title>` your HTML files with the name of the exercises.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by EuichangPark** [comment]
nodrink

Updated ones that I didn't do right.

File: javascript-if/main.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+function isOldEnoughToDrinkAndDrive(person) {
+  let result;
+  if (person.age < 999) {
+    result = false;
+  }
+  return result;
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js:66]: What about if they're older than that?

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by EuichangPark** [comment]
nodrink


fixed it!

File: javascript-if/main.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+function isOldEnoughToDrive(person) {
+  return false;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  let result;
+  if (person.age < 999) {
+    result = false;
+  }
+  return result;
```
Comment (Zhiwutian) [inline javascript-if/main.js:59]: this function should just return false.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update needed!

File: javascript-if/main.js (hunk 5 above)
Comment (EuichangPark) [reply javascript-if/main.js:59]: Oops I updated the wrong function... I have a change another one.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**